popup.js

```diff
@@ -0,0 +1,191 @@
+  document.getElementById('technology2').innerHTML = projectInformation[2].technologies.technology2;
+  document.getElementById('technology3').innerHTML = projectInformation[2].technologies.technology3;
+});
+const fourthProject = document.querySelector('.projectFour');
+fourthProject.addEventListener('click', () => {
+  document.getElementById('modal-title').innerHTML = projectInformation[3].name;
+  document.getElementById('description').innerHTML = projectInformation[3].description;
+  document.getElementById('featuredImage').innerHTML = projectInformation[3].featuredImage;
+  document.getElementById('technology1').innerHTML = projectInformation[3].technologies.technology1;
+  document.getElementById('technology2').innerHTML = projectInformation[3].technologies.technology2;
+  document.getElementById('technology3').innerHTML = projectInformation[3].technologies.technology3;
+});
+const fifthProject = document.querySelector('.projectFive');
+fifthProject.addEventListener('click', () => {
+  document.getElementById('modal-title').innerHTML = projectInformation[4].name;
+  document.getElementById('description').innerHTML = projectInformation[4].description;
+  document.getElementById('featuredImage').innerHTML = projectInformation[4].featuredImage;
+  document.getElementById('technology1').innerHTML = projectInformation[4].technologies.technology1;
+  document.getElementById('technology2').innerHTML = projectInformation[4].technologies.technology2;
+  document.getElementById('technology3').innerHTML = projectInformation[4].technologies.technology3;
+});
+const sixthProject = document.querySelector('.projectSix');
+sixthProject.addEventListener('click', () => {
+  document.getElementById('modal-title').innerHTML = projectInformation[5].name;
+  document.getElementById('description').innerHTML = projectInformation[5].description;
+  document.getElementById('featuredImage').innerHTML = projectInformation[5].featuredImage;
+  document.getElementById('technology1').innerHTML = projectInformation[5].technologies.technology1;
+  document.getElementById('technology2').innerHTML = projectInformation[5].technologies.technology2;
+  document.getElementById('technology3').innerHTML = projectInformation[5].technologies.technology3;
+});
+const seventhProject = document.querySelector('.projectSeven');
+seventhProject.addEventListener('click', () => {
+  document.getElementById('modal-title').innerHTML = projectInformation[6].name;
+  document.getElementById('description').innerHTML = projectInformation[6].description;
+  document.getElementById('featuredImage').innerHTML = projectInformation[6].featuredImage;
+  document.getElementById('technology1').innerHTML = projectInformation[6].technologies.technology1;
+  document.getElementById('technology2').innerHTML = projectInformation[6].technologies.technology2;
+  document.getElementById('technology3').innerHTML = projectInformation[6].technologies.technology3;
+});
```

Comment:
- [ ] [OPTIONAL] Kindly use a function to avoid code repetition
> eg 

```
const populatemodal = (clanname, index) => {
    const seventhProject = document.querySelector(clanname);
seventhProject.addEventListener('click', () => {
  document.getElementById('modal-title').innerHTML = projectInformation[index].name;
  document.getElementById('description').innerHTML = projectInformation[index].description;
  document.getElementById('featuredImage').innerHTML = projectInformation[index].featuredImage;
  document.getElementById('technology1').innerHTML = projectInformation[index].technologies.technology1;
  document.getElementById('technology2').innerHTML = projectInformation[index].technologies.technology2;
  document.getElementById('technology3').innerHTML = ### projectInformation[index].technologies.technology3;
});
}
```